input/pagecontent/index.md

```diff
@@ -57,8 +59,8 @@ IHE uses the normative words: Shall, Should, and May according to [standards con
 
 #### Must Support
 
```

Comment:
not sure we need the must support concept. It is used in a normative IG, we are writing a handbook.